composer.json

```diff
@@ -31,7 +31,10 @@
     "autoload-dev": {
         "psr-4": {
             "Tests\\": "tests/"
-        }
+        },
+        "files": [
```

Comment:
Maybe we can put them on Unit/Eloquent instead of creating another high level folder